File: 2.Bolum/isareteoperator/go/main.go
package main

import "fmt"

func main() {
	var tut int = 888
	var adr *int
	adr = &tut

	fmt.Printf("Degiskenin adresi : %p\n", &tut)
	fmt.Printf("Degiskenin degeri : %d\n\n", tut)
	fmt.Printf("Isaretcinin degeri (adres) : %p\n", adr)
	fmt.Printf("Isaret ettigi deger : %d\n\n", *adr)
	*adr = 444
	fmt.Printf("Degiskenin yeni degeri : %d\n", tut)
	fmt.Printf("DEgiskenin adresi : %p\n", &tut)
}

File: 2.Bolum/isareteoperator/c/main.c
main()
{
    int tut, *adr;
    adr = &tut;
    tut = 888;

    printf("Degiskenin adresi : %p\n", &tut);
    printf("Degiskenin degeri : %d\n\n", tut);
    printf("Isaretcinin degeri (adres) : %p\n", adr);
    printf("Isaret ettigi deger : %d\n\n", *adr);
    *adr = 444;
    printf("Degiskenin yeni degeri : %d\n", tut);
    printf("DEgiskenin adresi : %p\n", &tut);
}